### PR TITLE
Adds detail that joining sessions is not supported in recording proxy mode

### LINKED ISSUE #3913: Attempting to join a session fails when using `session_recording: proxy`
**What happened**: When Teleport is configured with `session_recording: proxy`, trying to join an ongoing session from the web UI results in an error:

```console
disconnected
failed connecting to node 6fae23b1-4e81-43fd-b4d5-4ae47313a0b3. unable to locate node matching uuid-like target 6fae23b1-4e81-43fd-b4d5-4ae47313a0b3
```

These are the matching errors in the auth/proxy log:

```console
DEBU [AUTH]      ClientCertPool -> cert(example-main.gravitational.co issued by example-main.gravitational.co:262320076426870836491040846249583905555) auth/middleware.go:359
DEBU [AUTH]      ClientCertPool -> cert(example-main.gravitational.co issued by example-main.gravitational.co:311829698878423599335352548065297639638) auth/middleware.go:359
DEBU [AUTH:1]    Server certificate cert(df1e526c-1692-4da3-afcc-9134915901e0.example-main.gravitational.co issued by example-main.gravitational.co:262320076426870836491040846249583905555). auth/middleware.go:164
INFO             web.getSetssion(a2fb26cc-108b-446f-a7ad-8de30a6cac15) web/apiserver.go:1531
DEBU             [WEB] new terminal request for ns=, server=6fae23b1-4e81-43fd-b4d5-4ae47313a0b3, login=root, sid=a2fb26cc-108b-446f-a7ad-8de30a6cac15 web/apiserver.go:1423
DEBU [AUTH]      GetServers(4->4) in 1.043721ms. elapsed_fetch:197.522µs elapsed_filter:846.199µs user:gus@gravitational.com auth/auth_with_roles.go:567
INFO             [WEB] getting terminal to '&web.TerminalRequest{Server:"6fae23b1-4e81-43fd-b4d5-4ae47313a0b3", Login:"root", Term:session.TerminalParams{W:201, H:52}, SessionID:"a2fb26cc-108b-446f-a7ad-8de30a6cac15", Namespace:"default", ProxyHostPort:"0.0.0.0:3080,3023", Cluster:"example-main.gravitational.co", InteractiveCommand:[]string(nil)}' web/apiserver.go:1442
DEBU [WEBSOCKET] Creating websocket stream for a2fb26cc-108b-446f-a7ad-8de30a6cac15. web/terminal.go:235
INFO [CLIENT]    Connecting proxy=0.0.0.0:3023 login='root' method=0 client/api.go:1623
DEBU [PROXY]     conn(127.0.0.1:43722->127.0.0.1:3023, user=root) auth attempt fingerprint:ssh-rsa-cert-v01@openssh.com SHA256:I+XqpgHyPL1BkZ4Fr4dqDONTlrAStElx6hcUc9KKY0c local:127.0.0.1:3023 remote:127.0.0.1:43722 user:root srv/authhandlers.go:151
DEBU [PROXY]     conn(127.0.0.1:43722->127.0.0.1:3023, user=root) auth attempt with key ssh-rsa-cert-v01@openssh.com SHA256:I+XqpgHyPL1BkZ4Fr4dqDONTlrAStElx6hcUc9KKY0c, &ssh.Certificate{Nonce:[]uint8{0x24, 0xb9, 0xf8, 0x5c, 0xbe, 0x8e, 0x14, 0x7, 0x32, 0x58, 0x19, 0x2, 0x71, 0xa7, 0xd4, 0x91, 0xc, 0xc8, 0xcd, 0xbf, 0x33, 0xa, 0x64, 0x2e, 0x1d, 0xbb, 0xaf, 0xb9, 0x4f, 0x3, 0x43, 0x7e}, Key:(*ssh.rsaPublicKey)(0xc0004890d0), Serial:0x0, CertType:0x1, KeyId:"gus@gravitational.com", ValidPrincipals:[]string{"root", "gus"}, ValidAfter:0x5ef4fd50, ValidBefore:0x5ef5a64c, Permissions:ssh.Permissions{CriticalOptions:map[string]string{}, Extensions:map[string]string{"permit-agent-forwarding":"", "permit-port-forwarding":"", "permit-pty":"", "teleport-roles":"{\"version\":\"v1\",\"roles\":[\"clusteradmin\"]}", "teleport-traits":"{\"aud\":[\"V0vOv0093JCBlGctSWA7vELnV0ufTazN\"],\"awsRole\":[\"arn:aws:iam::126027368216:role/auth0-admin,arn:aws:iam::126027368216:saml-provider/auth0\"],\"awsRoleSession\":[\"gus\"],\"clientID\":[\"V0vOv0093JCBlGctSWA7vELnV0ufTazN\"],\"created_at\":[\"2019-02-14T16:21:19.875Z\"],\"email\":[\"gus@gravitational.com\"],\"family_name\":[\"Luxton\"],\"given_name\":[\"Gus\"],\"iss\":[\"https://gravitational.auth0.com/\"],\"locale\":[\"en\"],\"login\":[\"gus.luxton\"],\"name\":[\"Gus Luxton\"],\"nickname\":[\"gus\"],\"picture\":[\"https://lh3.googleusercontent.com/a-/AOh14GibLqZlux5baIrCtDkPysX_uf00uwFuvQCHWbiY\"],\"roles\":[\"gravitational/admins\",\"gravitational/devc\",\"gravitational/wikireaders\"],\"sub\":[\"google-oauth2|102645889361203966163\"],\"unixaccountname\":[\"ec2-user\"],\"updated_at\":[\"2020-06-25T14:58:50.195Z\"],\"user_id\":[\"google-oauth2|102645889361203966163\"]}"}}, Reserved:[]uint8{}, SignatureKey:(*ssh.rsaPublicKey)(0xc000489130), Signature:(*ssh.Signature)(0xc001ba9780)} fingerprint:ssh-rsa-cert-v01@openssh.com SHA256:I+XqpgHyPL1BkZ4Fr4dqDONTlrAStElx6hcUc9KKY0c local:127.0.0.1:3023 remote:127.0.0.1:43722 user:root srv/authhandlers.go:151
DEBU [PROXY]     Successfully authenticated fingerprint:ssh-rsa-cert-v01@openssh.com SHA256:I+XqpgHyPL1BkZ4Fr4dqDONTlrAStElx6hcUc9KKY0c local:127.0.0.1:3023 remote:127.0.0.1:43722 user:root srv/authhandlers.go:196
DEBU [SSH:PROXY] Incoming connection 127.0.0.1:43722 -> 127.0.0.1:3023 vesion: SSH-2.0-Go. sshutils/server.go:430
INFO [CLIENT]    Successful auth with proxy 0.0.0.0:3023 client/api.go:1613
DEBU [PROXY]     Handling request subsystem, want reply true. id:17 local:127.0.0.1:3023 login:root remote:127.0.0.1:43722 teleportUser:gus@gravitational.com regular/sshserver.go:1122
DEBU [PROXY]     Subsystem request: proxySites(). id:17 local:127.0.0.1:3023 login:root remote:127.0.0.1:43722 teleportUser:gus@gravitational.com regular/sshserver.go:1236
DEBU [NODE]      proxysites.start(ServerContext(127.0.0.1:43722->127.0.0.1:3023, user=root, id=17)) regular/sites.go:55
DEBU [KEEPALIVE] Starting keep-alive loop with with interval 5m0s and max count 3. srv/keepalive.go:67
DEBU [NODE]      Subsystem proxySites() finished with result: <nil>. regular/sshserver.go:1244
DEBU [PROXY]     Close session request: <nil>. id:17 local:127.0.0.1:3023 login:root remote:127.0.0.1:43722 teleportUser:gus@gravitational.com regular/sshserver.go:1081
DEBU [CLIENT]    Found clusters: [{"name":"example-main.gravitational.co","lastconnected":"2020-06-25T19:41:01.141510866Z","status":"online"},{"name":"exampletwo-main.gravitational.co","lastconnected":"2020-06-25T19:37:00.337774653Z","status":"online"},{"name":"exampletwo-east.gravitational.co","lastconnected":"2020-06-25T19:37:00.369785322Z","status":"online"}] client/client.go:108
INFO [CLIENT]    Client=1.2.3.4:60416 connecting to node=6fae23b1-4e81-43fd-b4d5-4ae47313a0b3 on cluster example-main.gravitational.co client/client.go:554
DEBU [SSH:PROXY] Received out-of-band request: &{Type:recording-proxy@teleport.com WantReply:true Payload:[] ch:<nil> mux:0xc0002b6770}. sshutils/server.go:456
DEBU [NODE]      Global request (recording-proxy@teleport.com, true) received regular/sshserver.go:1284
DEBU [NODE]      Replied to global request (recording-proxy@teleport.com, true): true regular/sshserver.go:1306
DEBU [KEEPALIVE] Starting keep-alive loop with with interval 5m0s and max count 3. srv/keepalive.go:67
DEBU [PROXY]     Handling request env, want reply true. id:18 local:127.0.0.1:3023 login:root remote:127.0.0.1:43722 teleportUser:gus@gravitational.com regular/sshserver.go:1122
DEBU [PROXY]     Handling request auth-agent-req@openssh.com, want reply true. id:18 local:127.0.0.1:3023 login:root remote:127.0.0.1:43722 teleportUser:gus@gravitational.com regular/sshserver.go:1122
DEBU [PROXY]     Handling request subsystem, want reply true. id:18 local:127.0.0.1:3023 login:root remote:127.0.0.1:43722 teleportUser:gus@gravitational.com regular/sshserver.go:1122
DEBU [NODE]      parse_proxy_subsys("proxy:6fae23b1-4e81-43fd-b4d5-4ae47313a0b3:0@default@example-main.gravitational.co") regular/proxy.go:70
DEBU [NODE]      newProxySubsys({default 6fae23b1-4e81-43fd-b4d5-4ae47313a0b3 0 example-main.gravitational.co}). regular/proxy.go:172
DEBU [PROXY]     Subsystem request: proxySubsys(cluster=default/example-main.gravitational.co, host=6fae23b1-4e81-43fd-b4d5-4ae47313a0b3, port=0). id:18 local:127.0.0.1:3023 login:root remote:127.0.0.1:43722 teleportUser:gus@gravitational.com regular/sshserver.go:1236
DEBU [SUBSYSTEM] Starting subsystem trace.fields:map[dst:127.0.0.1:3023 src:127.0.0.1:43722] regular/proxy.go:201
DEBU [SUBSYSTEM] proxy connecting to host=6fae23b1-4e81-43fd-b4d5-4ae47313a0b3 port=0, exact port=false trace.fields:map[dst:127.0.0.1:3023 src:127.0.0.1:43722] regular/proxy.go:318
WARN [PROXY]     Subsystem request proxySubsys(cluster=default/example-main.gravitational.co, host=6fae23b1-4e81-43fd-b4d5-4ae47313a0b3, port=0) failed: unable to locate node matching uuid-like target 6fae23b1-4e81-43fd-b4d5-4ae47313a0b3. id:18 local:127.0.0.1:3023 login:root remote:127.0.0.1:43722 teleportUser:gus@gravitational.com regular/sshserver.go:1238
ERRO [NODE]      unable to locate node matching uuid-like target 6fae23b1-4e81-43fd-b4d5-4ae47313a0b3 regular/sshserver.go:1416
WARN [WEBSOCKET] Unable to stream terminal: failed connecting to node 6fae23b1-4e81-43fd-b4d5-4ae47313a0b3. unable to locate node matching uuid-like target 6fae23b1-4e81-43fd-b4d5-4ae47313a0b3. web/terminal.go:317
DEBU [WEBSOCKET] Closing websocket stream for a2fb26cc-108b-446f-a7ad-8de30a6cac15. web/terminal.go:241
DEBU [SSH:PROXY] Closed connection 127.0.0.1:43722. sshutils/server.go:432
```

`tsh -d join <session>` also fails:

```console
$ tsh -d join -l root 4c7a0fb6-b71b-11ea-bfb2-662a7f37020d
INFO [CLIENT]    [KEY AGENT] Connected to the system agent: "/tmp/ssh-BB9i0W69wikR/agent.4140" client/api.go:2109
DEBU [KEYSTORE]  Returning SSH certificate "/home/gus/.tsh/keys/example-main.gravitational.co/gus@gravitational.com-cert.pub" valid until "2020-06-26 02:57:56 -0300 ADT", TLS certificate "/home/gus/.tsh/keys/example-main.gravitational.co/gus@gravitational.com-x509.pem" valid until "2020-06-26 05:57:56 +0000 UTC". client/keystore.go:262
INFO [KEYAGENT]  Loading key for "gus@gravitational.com" client/keyagent.go:108
INFO [CLIENT]    Connecting proxy=example-main.gravitational.co:3023 login='root' method=0 client/api.go:1603
DEBU [KEYAGENT]  Validated host example-main.gravitational.co:3023. client/keyagent.go:280
INFO [CLIENT]    Successful auth with proxy example-main.gravitational.co:3023 client/api.go:1594
DEBU [CLIENT]    Found clusters: [{"name":"example-main.gravitational.co","lastconnected":"2020-06-25T19:43:37.105342623Z","status":"online"},{"name":"exampletwo-main.gravitational.co","lastconnected":"2020-06-25T19:42:00.054583736Z","status":"online"},{"name":"exampletwo-east.gravitational.co","lastconnected":"2020-06-25T19:42:00.191432387Z","status":"online"}] client/client.go:106
DEBU [KEYSTORE]  Adding trusted cluster certificate authority "SERIALNUMBER=262320076426870836491040846249583905555,CN=example-main.gravitational.co,O=example-main.gravitational.co" to trusted pool. client/keystore.go:328
DEBU [KEYSTORE]  Adding trusted cluster certificate authority "SERIALNUMBER=119246687608421557029362004674317167784,CN=exampletwo-main.gravitational.co,O=exampletwo-main.gravitational.co" to trusted pool. client/keystore.go:328
DEBU [KEYSTORE]  Returning SSH certificate "/home/gus/.tsh/keys/example-main.gravitational.co/gus@gravitational.com-cert.pub" valid until "2020-06-26 02:57:56 -0300 ADT", TLS certificate "/home/gus/.tsh/keys/example-main.gravitational.co/gus@gravitational.com-x509.pem" valid until "2020-06-26 05:57:56 +0000 UTC". client/keystore.go:262
DEBU [CLIENT]    Client  is connecting to auth server on cluster "example-main.gravitational.co". client/client.go:469

ERROR REPORT:
Original Error: *trace.NotFoundError session '4c7a0fb6-b71b-11ea-bfb2-662a7f37020d' not found or it has ended
Stack Trace:
    /gopath/src/github.com/gravitational/teleport/lib/client/api.go:1067 github.com/gravitational/teleport/lib/client.(*TeleportClient).Join
    /gopath/src/github.com/gravitational/teleport/tool/tsh/tsh.go:862 main.onJoin.func1
    /gopath/src/github.com/gravitational/teleport/lib/client/api.go:314 github.com/gravitational/teleport/lib/client.RetryWithRelogin
    /gopath/src/github.com/gravitational/teleport/tool/tsh/tsh.go:861 main.onJoin
    /gopath/src/github.com/gravitational/teleport/tool/tsh/tsh.go:340 main.Run
    /gopath/src/github.com/gravitational/teleport/tool/tsh/tsh.go:186 main.main
    /opt/go/src/runtime/proc.go:212 runtime.main
    /opt/go/src/runtime/asm_amd64.s:1358 runtime.goexit
User Message: session '4c7a0fb6-b71b-11ea-bfb2-662a7f37020d' not found or it has ended

```

Proxy/auth logs:

```console
DEBU [PROXY]     conn(1.2.3.4:42208->10.4.0.52:3023, user=root) auth attempt fingerprint:ssh-rsa-cert-v01@openssh.com SHA256:opmZqD5yENtb8772Irkez+BIomCMMth5DE1paxPjJ4g local:10.4.0.52:3023 remote:1.2.3.4:42208 user:root srv/authhandlers.go:151
DEBU [PROXY]     conn(1.2.3.4:42208->10.4.0.52:3023, user=root) auth attempt with key ssh-rsa-cert-v01@openssh.com SHA256:opmZqD5yENtb8772Irkez+BIomCMMth5DE1paxPjJ4g, &ssh.Certificate{Nonce:[]uint8{0x1, 0xac, 0x12, 0x7e, 0x72, 0xda, 0x67, 0x9b, 0x54, 0x74, 0x61, 0x7f, 0xfa, 0x47, 0x56, 0x28, 0x66, 0xa2, 0x5a, 0x22, 0x73, 0xa4, 0x7a, 0x9d, 0xa5, 0x96, 0x1b, 0x6f, 0xda, 0x35, 0xe4, 0x9}, Key:(*ssh.rsaPublicKey)(0xc001a31f40), Serial:0x0, CertType:0x1, KeyId:"gus@gravitational.com", ValidPrincipals:[]string{"root", "gus"}, ValidAfter:0x5ef4e568, ValidBefore:0x5ef58e64, Permissions:ssh.Permissions{CriticalOptions:map[string]string{}, Extensions:map[string]string{"permit-agent-forwarding":"", "permit-port-forwarding":"", "permit-pty":"", "teleport-roles":"{\"version\":\"v1\",\"roles\":[\"clusteradmin\"]}", "teleport-traits":"{\"aud\":[\"V0vOv0093JCBlGctSWA7vELnV0ufTazN\"],\"awsRole\":[\"arn:aws:iam::126027368216:role/auth0-admin,arn:aws:iam::126027368216:saml-provider/auth0\"],\"awsRoleSession\":[\"gus\"],\"clientID\":[\"V0vOv0093JCBlGctSWA7vELnV0ufTazN\"],\"created_at\":[\"2019-02-14T16:21:19.875Z\"],\"email\":[\"gus@gravitational.com\"],\"family_name\":[\"Luxton\"],\"given_name\":[\"Gus\"],\"iss\":[\"https://gravitational.auth0.com/\"],\"locale\":[\"en\"],\"login\":[\"gus.luxton\"],\"name\":[\"Gus Luxton\"],\"nickname\":[\"gus\"],\"picture\":[\"https://lh3.googleusercontent.com/a-/AOh14GibLqZlux5baIrCtDkPysX_uf00uwFuvQCHWbiY\"],\"roles\":[\"gravitational/admins\",\"gravitational/devc\",\"gravitational/wikireaders\"],\"sub\":[\"google-oauth2|102645889361203966163\"],\"unixaccountname\":[\"ec2-user\"],\"updated_at\":[\"2020-06-25T17:57:55.253Z\"],\"user_id\":[\"google-oauth2|102645889361203966163\"]}"}}, Reserved:[]uint8{}, SignatureKey:(*ssh.rsaPublicKey)(0xc001a31f80), Signature:(*ssh.Signature)(0xc000785ac0)} fingerprint:ssh-rsa-cert-v01@openssh.com SHA256:opmZqD5yENtb8772Irkez+BIomCMMth5DE1paxPjJ4g local:10.4.0.52:3023 remote:1.2.3.4:42208 user:root srv/authhandlers.go:151
DEBU [PROXY]     Successfully authenticated fingerprint:ssh-rsa-cert-v01@openssh.com SHA256:opmZqD5yENtb8772Irkez+BIomCMMth5DE1paxPjJ4g local:10.4.0.52:3023 remote:1.2.3.4:42208 user:root srv/authhandlers.go:196
DEBU [SSH:PROXY] Incoming connection 1.2.3.4:42208 -> 10.4.0.52:3023 vesion: SSH-2.0-Go. sshutils/server.go:430
DEBU [KEEPALIVE] Starting keep-alive loop with with interval 5m0s and max count 3. srv/keepalive.go:67
DEBU [PROXY]     Handling request subsystem, want reply true. id:21 local:10.4.0.52:3023 login:root remote:1.2.3.4:42208 teleportUser:gus@gravitational.com regular/sshserver.go:1122
DEBU [PROXY]     Subsystem request: proxySites(). id:21 local:10.4.0.52:3023 login:root remote:1.2.3.4:42208 teleportUser:gus@gravitational.com regular/sshserver.go:1236
DEBU [NODE]      proxysites.start(ServerContext(1.2.3.4:42208->10.4.0.52:3023, user=root, id=21)) regular/sites.go:55
DEBU [NODE]      Subsystem proxySites() finished with result: <nil>. regular/sshserver.go:1244
DEBU [PROXY]     Close session request: <nil>. id:21 local:10.4.0.52:3023 login:root remote:1.2.3.4:42208 teleportUser:gus@gravitational.com regular/sshserver.go:1081
DEBU [KEEPALIVE] Starting keep-alive loop with with interval 5m0s and max count 3. srv/keepalive.go:67
DEBU [PROXY]     Handling request subsystem, want reply true. id:22 local:10.4.0.52:3023 login:root remote:1.2.3.4:42208 teleportUser:gus@gravitational.com regular/sshserver.go:1122
DEBU [NODE]      parse_proxy_subsys("proxy:@example-main.gravitational.co") regular/proxy.go:70
DEBU [NODE]      newProxySubsys({default   example-main.gravitational.co}). regular/proxy.go:172
DEBU [PROXY]     Subsystem request: proxySubsys(cluster=default/example-main.gravitational.co, host=, port=). id:22 local:10.4.0.52:3023 login:root remote:1.2.3.4:42208 teleportUser:gus@gravitational.com regular/sshserver.go:1236
DEBU [SUBSYSTEM] Starting subsystem trace.fields:map[dst:10.4.0.52:3023 src:1.2.3.4:42208] regular/proxy.go:201
INFO [SUBSYSTEM] Connected to auth server: 10.4.0.52:3025 trace.fields:map[dst:10.4.0.52:3023 src:1.2.3.4:42208] regular/proxy.go:253
DEBU [AUTH]      ClientCertPool -> cert(example-main.gravitational.co issued by example-main.gravitational.co:262320076426870836491040846249583905555) auth/middleware.go:359
DEBU [AUTH]      ClientCertPool -> cert(exampletwo-east.gravitational.co issued by exampletwo-east.gravitational.co:211285690612341962132715856004605634477) auth/middleware.go:359
DEBU [AUTH]      ClientCertPool -> cert(exampletwo-main.gravitational.co issued by exampletwo-main.gravitational.co:119246687608421557029362004674317167784) auth/middleware.go:359
DEBU [AUTH]      ClientCertPool -> cert(example-main.gravitational.co issued by example-main.gravitational.co:311829698878423599335352548065297639638) auth/middleware.go:359
DEBU [AUTH:1]    Server certificate cert(df1e526c-1692-4da3-afcc-9134915901e0.example-main.gravitational.co issued by example-main.gravitational.co:262320076426870836491040846249583905555). auth/middleware.go:164
DEBU [SSH:PROXY] Closed connection 1.2.3.4:42208. sshutils/server.go:432
DEBU [NODE]      Subsystem proxySubsys(cluster=default/example-main.gravitational.co, host=, port=) finished with result: read tcp 10.4.0.52:51600->10.4.0.52:3025: use of closed network connection. regular/sshserver.go:1244
DEBU [PROXY]     Client 1.2.3.4:42208 disconnected. id:22 local:10.4.0.52:3023 login:root remote:1.2.3.4:42208 teleportUser:gus@gravitational.com regular/sshserver.go:1086
```

Changing back to `session_recording: node` makes everything work as expected.

**What you expected to happen**: Joining a session when using proxy recording mode should work as expected.

**How to reproduce it (as minimally and precisely as possible)**: Set `session_recording: proxy`, start a session, try to join.

### Environment
- Teleport version (use `teleport version`): `Teleport Enterprise v4.3.0-master.1git:v4.2.0-alpha.5-444-g3976a9977-dirty go1.13.2`

### PR DESCRIPTION
**Pull request by webvictim**
Fixes #3913 

Also fixes automatic `milv` runs in Drone, and then fixes all the stuff that `milv` complained about... @benarent 

Also includes a few other miscellaneous fixes to `.drone.yml` which needed to be made.

### CODE REVIEW

File: docs/4.3/user-manual.md [hunk 1]
```diff
@@ -439,6 +439,10 @@ can join you through her terminal by typing:
 $ tsh join <session_ID>
 ```
 
+!!! note
+
+Joining sessions is not supported in recording proxy mode (where `session_recording` is set to `proxy`).
```
Comment (benarent) [inline docs/4.3/user-manual.md:444]: Don't these also need to be indented? Regardless, we should try to keep to one style for these.

**Review by benarent — APPROVED** [review]
Thanks for cleaning up Milv :-) I was going to wait until the end, but good to know drone tests are running now.

File: docs/4.3/user-manual.md (hunk 1 above)
Comment (webvictim) [reply docs/4.3/user-manual.md]: You're right, good catch. Fixed.

**Comment by webvictim** [comment]
@russjones Can I get a rubber stamp please?

**Review by russjones — APPROVED** [review]

**Pull request merged**